docs/antora-site-structure.md

```diff
@@ -31,41 +31,56 @@ The main reasons for using [Antora][link-antora] is the following:
 
 ## Scope of Documentation Repositories
 
-The ownCloud documentation consists of a master repo named `docs` which includes additional product repos like the clients or others (content sources defined in `site.yml`). While each repo can be built individually for testing, only the build of docs creates documentation which is pushed to the web.
+The ownCloud documentation consists of a main repo named `docs` and additional product repos like the clients or others which are included as content sources defined in `site.yml`.
+
+Our setup is made in a way, where each repo can build it's own content individually for testing and checking validity. Only the build of main repo `docs` creates documentation which is pushed to the web. There is one exeption, pdf's are created AND pushed via the CI from _each_ content source.
+
 
 ```
-layer-1     layer-2
-master      included repos
+main        content source
 
             docs-client-desktop
 docs    --> docs-client-ios-app 
             docs-client-android
             ...
 
 ```
-Note that the arrow from level-1 to level-2 is intentionally unidirectional and should be respected. See more details about the reason below. 
+Note that the arrow from main to content source is intentionally unidirectional in our setup and should be respected. See more details about the reason below. 
+
+### Scope of the playbook files (site.yml)
+
+In general, only one `site.yml` file is neccessary for the whole environment and its definitions are **available to the whole site**. This `site.yml` file is located in the main repo `docs`. We have additionally for each content source its own `site.yml` for testing purposes only. The scope of these local `site.yml` files is restricted to the respective content source and any definitions made are not availabe outside. 
+
+Due to this fact, you need to re-add all relevant attributes of the main `site.yml` file in the `site.yml` file of the content source which accesses it, else a local build will return warnings about unresolved attributes.
+
+If you have added an attribute in a content source `site.yml` file, you must add this attibute to the main `site.yml` file to avoid a build warning (unresolved attribute) during a build of the entire documentation.
+
+Note that this behaviour is relevant for the playbook `site.yml` files only and does not apply to the component descriptor files `antora.yml`.
 
 ## Scope of Content Accessibility
 
-Because Antora is capable of defining additional resources, you can access content from these resources. To do so, follow the [Resource ID Coordinates][link-resource-id] scheme.
+Because Antora is capable of defining additional content sources, you can access content from these resources. The setup is flat, no main/child environment. To access ressources, follow the [Resource ID Coordinates][link-resource-id] scheme.
```

Comment:
```suggestion
Because Antora is capable of defining additional content sources, you can access content from these resources. The setup is flat, no main/child environment. To access resources, follow the [Resource ID Coordinates][link-resource-id] scheme.
```